pkg/rtc/participant.go

```diff
@@ -781,7 +793,8 @@ func (p *ParticipantImpl) GetSubscribedTracks() []types.SubscribedTrack {
 // AddSubscribedTrack adds a track to the participant's subscribed list
 func (p *ParticipantImpl) AddSubscribedTrack(subTrack types.SubscribedTrack) {
 	p.params.Logger.Debugw("added subscribedTrack",
-		"publisher", subTrack.PublisherIdentity(),
+		"publisherID", subTrack.PublisherID(),
+		"publisherIdentity", subTrack.PublisherIdentity(),
```

Comment:
Still using the `PublisherIdentity` just for logging. This and the log below are the only places where `PublisherIdentity` of `SubscribedTrack` is used.